scripts/create-seed-fixture-heartwood

```diff
@@ -1,24 +1,24 @@
 #!/usr/bin/env bash
+
+killall radicle-node
+
 set -euo pipefail
 
-PASSPHRASE=asdf
-SEED="0.0.0.0:8080"
+export RAD_PASSPHRASE=asdf
 
 REPO_ROOT=$(git rev-parse --show-toplevel)
-ID=$(echo $RANDOM | md5sum | head -c 8)
-BASE_PATH=$REPO_ROOT/tests/tmp/create-seed-fixture-$ID
+ID=$(echo $RANDOM | md5sum | head -c 4)
```

Comment:
There's a limit to SUN_LEN (len of path to socket) so I made heartwood path shorter.